api/v1/cluster_types.go

```diff
@@ -0,0 +1,80 @@
+package v1
+
+const (
+	// Ray node state.
+
+	DeadNodeState  = "DEAD"
+	AliveNodeState = "ALIVE"
+
+	// NeutreeServingVersionLabel is the label key of Neutree Serving version.
+	NeutreeServingVersionLabel = "neutree.ai/neutree-serving-version"
+
+	// NeutreeNodeProvisionTypeLabel is the label key of Neutree node provision type.
+	// It can be either "static" or "autoscaler".
+	NeutreeNodeProvisionTypeLabel = "neutree.ai/node-provision-type"
```

Comment:
What is the standard for placing fields in label or spec/status?

Reply:
The real cluster status should place in status, spec is only record user input, and label may be managed by others but not neutree.